m03_acos/create_acos_exercises.rst

```diff
@@ -0,0 +1,454 @@
+  Requirements:
+    #. You need intermediate to advanced computing, and programming skills.
+    #. Prior knowledge about npm packages, JSON, HTML and javascript may be beneficial.
+    #. :download:`Download  </course_material/acos_drag_and_drop/Acos-drag-and-drop-activity.zip>` the images and code files used in this chapter.
+
+  Estimated working time:
+    60-90 min. 
+
+Introduction
+------------
+Acos exercises are basically an `npm package <https://docs.npmjs.com/about-packages-and-modules>`_ which, if desired, can be published in the npm package manager environment or simply used privately in your own course. Acos exercises are also referred to as "Acos content packages" in the :ref:`Acos environment <acos-concept>`. These Acos content packages are basically a container for the Acos exercises. Therefore, in this chapter, we will show you how to create a drag and drop exercise. Notwithstanding, the  concepts and ideas should be reusable to create other types of Acos exercises.
+
+.. figure:: /images/gallery/acos_concept.jpg
+  :name: acos-concept
+  :width: 60 %
+  :align: center
+  :class: img-responsive img-thumbnail
+
+  Acos server Team, (2016), **Acos architecture** [ONLINE]. Available at: https://www.npmjs.com/package/acos-server [Accessed 13 May 2020].
+
+.. _create-acos-exercise:
+
+Create an Acos exercise
+-----------------------
+Usability aspects
+.................
+Create an exercise is relatively easy. Basically, you only need to create an HTML file with the content you would like to have and a JSON configuration file with the `drag and drop configurations <https://git.io/JfEQW>`_. However, the most challenging part is to design the flow and the appearance of your exercises. 
+
+.. hint:: If you want to find inspiration, you can review some of the `exercises <https://acos.cs.aalto.fi/>`_  that has been used in some of the A+ courses. 
+
+Unfortunately, we cannot give you suggestions about the usability and the optimal flow of your exercises, yet we can provide you with a list of tools that may help improve the appearance of the Acos exercises. 
+
+first, we suggest using a CSS framework for styling your activities such as `Bootstrap <https://getbootstrap.com/>`_, `Material <https://material.io/>`_, or `Foundation <https://get.foundation/>`_. Secondly, we suggest making use of the `style editor <https://developer.mozilla.org/en-US/docs/Tools/Style_Editor>`_ embedded in your web browser. Finally, we suggest researching about `usability <https://www.nngroup.com/articles/usability-101-introduction-to-usability/>`_, especially usability related to the exercise you are developing. In this case, we should read about the main considerations needed while using `drag and drop <https://uxdesign.cc/drag-and-drop-for-design-systems-8d40502eb26d>`_ elements.
+
+.. rst-class:: pull-right 
+
+:glyphicon-info-sign:`\ ` **Read more about**  `HOW TO use the web inspector <https://developer.mozilla.org/en-US/docs/Tools/Page_Inspector/How_to>`_
```

Comment:
":glyphicon-info-sign:" appears to me as just the text ":glyphicon-info-sign:" when reading with the web browser (using local host). Does the icon actually work?